libsolidity/ast/AST.h

```diff
@@ -2625,6 +2625,37 @@ class Builtin: public Expression
 	SourceLocation m_nameParameterLocation;
 };
 
+class ForAllQuantifier: public ASTNode, public Scopable, public ScopeOpener
```

Comment:
Should I call it like this or would something like `UniversalQuantifier` be a better name? Any other suggestions?

Reply:
Fine as is.